README.md

```diff
@@ -1,20 +1,31 @@
-# Hotline-webring
+# Hotline Webring
 
-## Getting Started
+[This can only mean one thing!][video]
+
+[video]: https://www.youtube.com/watch?v=uxpDa-c-4Mc
+
+Q: No seriously, what is it?
+A: Way back in the 1990s, there were no search engines, but people still wanted
+   that sweet sweet traffic. And thus the webring: a group (or ring) of websites
+   with links to other websites in the ring.
 
-After you have cloned this repo, run this setup script to set up your machine
-with the necessary dependencies to run and test this app:
+Q: So it's some weird old thing?
+A: Um it's [vintage, and so adorable](https://www.youtube.com/watch?v=kfLSjobM9bg).
 
-    % ./bin/setup
+Q: How do I get in on this?
+A: Make up an identifier for yourself like `gabebw`. Then add links back and
+   forward in the ring: `URL.com/gabebw/previous` and `URL.com/gabebw/next`.
+   You'll be automatically added to the ring.
```

Comment:
The truly lazy should use HTML entities: 3ffe3746b8fae8e1e5a9cb461044aa4babefca10